libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/WritePropertiesEntitlement.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.entitlement.runtime.policy;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * An Entitlement to allow writing properties such as system properties.
+ */
+public record WritePropertiesEntitlement(Set<String> properties) implements Entitlement {
```

Comment:
Can we be more explicit in the name? These are _system_ properties (this does not affect the Properties class).

Reply:
The SecurityManager's PropertyPermission covers system properties, but also a bunch of random java.util defaults (Timezone, Locale). Initially, I was planning to use the same entitlements for those. But giving those a 2nd thought, we probably want to forbid setting those at runtime anyways, right? So renaming makes sense 👍

Reply:
++

Reply:
renamed 👍